low-level/framework/base/sspl_constants.py

```diff
@@ -226,6 +226,13 @@
     "storage_type_supported": ["rbod", "jbod", "ebod"]
 }
 
+EXCLUDED_SERVICES = {
```

Comment:
Needs to be in sspl.conf to keep it configurable. Manage under services section

Reply:
Done. Please review.